src/ec.rs

```diff
@@ -193,22 +193,18 @@ impl StorageBackend for WriterBase<WriterCounter> {
   fn store(&mut self, fl: u16, fh: u16, nms: u16) {
     let (_l, r) = self.lr_compute(fl, fh, nms);
     let d = 16 - ILog::ilog(r);
-    let mut s = self.cnt + (d as i16);
-
-    self.s.bytes += (s >= 0) as usize + (s >= 8) as usize;
-    s -= 8 * ((s >= 0) as i16 + (s >= 8) as i16);
 
+    self.s.bits += d;
     self.rng = r << d;
-    self.cnt = s;
   }
   #[inline]
-  fn stream_bytes(&mut self) -> usize {
-    self.s.bytes
+  fn stream_bits(&mut self) -> usize {
+    self.s.bits
   }
   #[inline]
   fn checkpoint(&mut self) -> WriterCheckpoint {
     WriterCheckpoint {
-      stream_bytes: self.s.bytes,
+      stream_bytes: self.s.bits,
```

Comment:
Can we rename stream_bytes, maybe to e.g. stream_counter? It is convenient that both Counter and Recorder use the same Checkpoint type I suppose, but at least a rename and comment would make it less confusing.